integrations/docker/images/chip-build-efr32/Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN set -x \
     gcc-arm-none-eabi=15:9-2019-q4-0ubuntu1 \
     binutils-arm-none-eabi=2.34-4ubuntu1+13ubuntu1 \
     ccache=3.7.7-1 \
+    apt-get install git-lfs \
```

Comment:
We need to also compile in vscode images. please make the following changes:

- add git-lfs to chip-build image instead of just for efr32
- also update version file to make the docker file version different.

Reply:
done